src/main/kotlin/bandcampcollectiondownloader/core/BandcampCollectionDownloader.kt

```diff
@@ -177,11 +179,23 @@ class BandcampCollectionDownloader(private val args: Args, private val io: IO) {
         // Replace invalid chars by similar unicode chars
         releasetitle = Util.replaceInvalidCharsByUnicode(releasetitle)
         artist = Util.replaceInvalidCharsByUnicode(artist)
+        bandName = Util.replaceInvalidCharsByUnicode(bandName)
 
         // Prepare artist and release folder paths
         val downloadFolderPath = Paths.get("${args.pathToDownloadFolder}")
-        val releaseFolderName = "$releaseYear - $releasetitle"
-        var artistFolderPath = downloadFolderPath.resolve(artist)
+
+        var releaseFolderName: String?
+        var artistFolderPath: Path?
+
+        if (!args.useBandName) {
+            artistFolderPath = downloadFolderPath.resolve(artist)
+            releaseFolderName = "$releaseYear - $releasetitle"
+        } else {
+            artistFolderPath = downloadFolderPath.resolve(bandName)
+            releaseFolderName = "$releaseYear - $artist - $releasetitle"
```

Comment:
Why the `$artist` here in the folder name when `useBandName` is `false`? Any specific reason? Just being curious.

Reply:
seems odd I'll check it out